lib/sidekiq_smart_cache/redis.rb

```diff
@@ -28,17 +31,22 @@ def log_msg(msg) # WIP all this
 
     def method_missing(name, *args)
       @pool.with do |r|
-        if r.respond_to?(name)
+        if COMMANDS.include? name
           retryable = true
           begin
             # log_msg("#{name} #{args}")
-            r.send(name, *args)
```

Comment:
I'm wondering if any of the COMMANDS accept a mixture of positional and keyword arguments?  If so, I anticipate running into issues in Ruby 3 on line 41 because passing `*args` as the second argument in call is banking on conversion of hash into keyword args (assuming keyword args are accepted).  My plan for the method_missing definitions with similar logic in Indigo is to to use argument forwarding with a leading argument (https://actblue.atlassian.net/browse/CORE-1116)...which should work for line 41, but wouldn't work with the logic on line 39 here.